squarelet/oidc/rules.py

```diff
@@ -13,7 +13,7 @@ def is_owner(user, client):
     return user == client.owner
 
 
-add_perm("oidc_provider.view_client", is_authenticated & is_owner)
-add_perm("oidc_provider.add_client", is_authenticated)
-add_perm("oidc_provider.change_client", is_authenticated & is_owner)
-add_perm("oidc_provider.delete_client", is_authenticated & is_owner)
+add_perm("oidc_provider.view_client", is_staff)
```

Comment:
So only staff will create clients, but do you still want users to be able to view a list of their own clients?

Reply:
Not at the moment, though I could imagine exposing that at some point.